Brewfile

```diff
@@ -0,0 +1,6 @@
+brew 'postgresql@13'
```

Comment:
somehow I was on postgres 14 (which I think is what matches prod right?)

Reply:
Thanks for trying it out, @zachmargolis ! 🎉  I'm glad it worked for you.

We are actually using [postgres@13 in production](https://github.com/18F/identity-devops/blob/33ba02736b37f3a79c50479892a03c6f3e920041/terraform/app/rds-variables.tf#L82). In [the Slack thread regarding this ticket](https://gsa-tts.slack.com/archives/C0NGESUN5/p1684521612583589?thread_ts=1684517846.543419&cid=C0NGESUN5), it was mentioned that we may be moving to 14 in the coming months but are not there yet.